### DISCUSSION TITLE
Bundled LiteLLM support has been deprecated from 0.2.0

### DISCUSSION THREAD

**Discussion by ole100** [open-webui/open-webui#2912] [General]
Hello everyone,
 I found can not use LiteLLM anymore in latest version, how can I use gemini with LiteLLM in the latest version, I still want this.
 

Thanks

**Comment by justinh-rahb** [comment]
Two options:
1. https://docs.openwebui.com/migration#migrating-from-internal-to-external-litellm
2. https://github.com/open-webui/pipelines

We'd prefer the 2nd ourselves but if you've already got a LiteLLM config, the first is easiest.
